### ISSUE TITLE
Feature Request- load settings from print settings profile in part settings

### ISSUE THREAD

**Issue by pyrophreek** [alexrj/Slic3r#2362]
I have been working with a kraken quad extruder and doing a lot of work on multi-part single prints using different materials and extruders (ie. dif nozzle sizes, combining paste and filament etc) and having to modify all of the settings per part each time is becoming tedious.  It would be nice to be able to load a print setting from a profile for each different component rather than have to manually change each of the settings individually.

PS: Thank you for all of the hard work.

**Comment by sirkha** [comment]
Yes, replacing parts would be really great. Either by right clicking on it and clicking replace part or with a button.

**Comment by alexrj** [comment]
Plates, along with per-object and per-part settings, can be exported and imported in the AMF format.
Profile settings can be exported and imported with *File -> Export config* and *File -> Load config*.

What do you mean by replacing parts? What's wrong with deleting one object in plater and adding the new one? (Or doing the same thing inside the object part editor if you want to keep per-object settings.)

@pyrophreek, if you experience a crash can you help us to fix it by reporting a reproducible test case? (in a separate issue please)

**Comment by sirkha** [comment]
if you delete a part and re-add it you have to redo all the print settings you have for that part. As pyrophreek says, it is common to update a part in a multimaterial print, and want to keep the print settings for that part, without having to re-select them all.

**Comment by alexrj** [comment]
@sirkha, that's a matter of reloading the same part then. It's tracked in #2473

**Comment by sirkha** [comment]
@alexrj I am not sure how #2473 is applicable. I am talking about one portion of a multi-material part with its individual per-part settings. The combined part contains multiple stls as parts and modifiers, if you make a change to one in your modeling program, it would be very convienient to be able to put the updated stl into the combined part without having to delete the old one and readding it. Additionally, as far as I can tell, there is no way to change the order, so if one part changes, you really have to re add all the sub-parts and modifiers to get the same combined part.
